### ISSUE TITLE
Eigen is a broken dependency

### ISSUE THREAD

**Issue by tkphd** [usnistgov/ChebTools#22]
```
$ git clone --recursive https://github.com/usnistgov/ChebTools
Cloning into 'ChebTools'...
remote: Counting objects: 707, done.
remote: Total 707 (delta 0), reused 0 (delta 0), pack-reused 707
Receiving objects: 100% (707/707), 1.01 MiB | 0 bytes/s, done.
Resolving deltas: 100% (359/359), done.
Submodule 'externals/Catch' (https://github.com/philsquared/Catch.git) registered for path 'externals/Catch'
Submodule 'externals/Eigen' (https://github.com/RLovelett/eigen.git) registered for path 'externals/Eigen'
Submodule 'externals/pybind11' (https://github.com/pybind/pybind11.git) registered for path 'externals/pybind11'
Cloning into 'ChebTools/externals/Catch'...
remote: Counting objects: 20151, done.        
remote: Compressing objects: 100% (124/124), done.        
remote: Total 20151 (delta 78), reused 109 (delta 50), pack-reused 19973        
Receiving objects: 100% (20151/20151), 9.78 MiB | 16.48 MiB/s, done.
Resolving deltas: 100% (13584/13584), done.
Cloning into 'ChebTools/externals/Eigen'...
Username for 'https://github.com': ^C
warning: Clone succeeded, but checkout failed.
You can inspect what was checked out with 'git status'
and retry the checkout with 'git checkout -f HEAD'
```

**Comment by ianhbell** [comment]
Doh.  Eigen have added a github mirror of their repo and this is now broken.  Let me fix it right now.

**Comment by ianhbell** [comment]
I think I fixed it - please let me know if not

**Comment by tkphd** [comment]
Yep, fixed by 719eb52bb6!
